src/main/java/de/javakaffee/kryoserializers/guava/ImmutableMapSerializer.java

```diff
@@ -54,5 +54,17 @@ public static void registerSerializers(final Kryo kryo) {
         kryo.register(ImmutableMap.of(o1, o1).getClass(), serializer);
         kryo.register(ImmutableMap.of(o1, o1, o2, o2).getClass(), serializer);
 
+        Map<DummyEnum,Object> enumMap = new EnumMap<DummyEnum, Object>(DummyEnum.class);
+        for (DummyEnum e : DummyEnum.values()) {
+            enumMap.put(e, o1);
```

Comment:
Is this really needed? I'd say it's enough to have an `EnumMap` instance, even if it's empty.